pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/ConsentElection.java

```diff
@@ -3,37 +3,22 @@
 import javax.validation.constraints.NotBlank;
 
 import com.google.gson.annotations.SerializedName;
+import lombok.Data;
 import org.broadinstitute.ddp.util.MiscUtil;
 
-public class ConsentElection {
-
+@Data
+public final class ConsentElection {
     @NotBlank
     @SerializedName("stableId")
-    private String stableId;
+    private final String stableId;
 
     @SerializedName("selected")
     private Boolean selected;
 
-    private transient String selectedExpr;
+    private final transient String selectedExpr;
 
     public ConsentElection(String stableId, String selectedExpr) {
         this.stableId = MiscUtil.checkNotBlank(stableId, "stableId");
         this.selectedExpr = MiscUtil.checkNotBlank(selectedExpr, "selectedExpr");
     }
```

Comment:
This class does not seem to have any relation to the PR- why are these modifications being included here?

Reply:
It looks like I had some Lombok-related changes and forgot to stash them. If it is a big deal, then I'll revert this change.